ASM/src/hacks.asm

```diff
@@ -2600,11 +2600,18 @@ skip_bombchu_bowling_prize_switch:
 ; Trade Quest Shuffle Flag Hooks
 ;==================================================================================================
 ; Control if Fado (blonde Kokiri girl) can spawn in Lost Woods
-.orga 0xE538C4
-    or      t3, $zero, $ra
+.headersize (0x80AD1C20 - 0xE51A60)
+.org 0x80AD3A84    ; Fado branch at end of EnKo_CanSpawn (called by EnKo_Init), override to always true
+    addiu   t5, $zero, 0x0031
+.org 0x80AD3EB0    ; end of EnKo_Init, controls Fado spawn
```

Comment:
Since you moved this hack to the end of EnKo_Init, can this new hack inadvertently prevent spawning other kokiri? I don't see anything in check_fado_spawn_flags actually checking that the actor being spawned is in fact fado

Reply:
I added a check for Fado's actor type. Vanilla behavior would kill all EnKo actors in Lost Woods. I changed it to only if it is Fado's actor. Her behavior elsewhere is unchanged (to my knowledge only the end of the bridge in Kokiri Forest).

Reply:
Sorry, I did pick up on that and test for it on my end. Glad it seems to be working on your end too.